app/Http/Controllers/Api/LicenseSeatsController.php

```diff
@@ -48,11 +48,11 @@ public function index(Request $request, $licenseId)
             }
 
             $limit = app('api_limit_value');
-
+            $page = $request->get('page');
```

Comment:
We don't actually send paging through though - that's something we've talked about in the past (versus making API users figure out offsets and such)

Reply:
if I remove $page from the transformer it breaks`$seat_count`